src/transformers/processing_utils.py

```diff
@@ -106,6 +108,9 @@ class TextKwargs(TypedDict, total=False):
             The side on which padding will be applied.
     """
 
+    text_pair: Optional[Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]]]
+    text_target: Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]]
+    text_pair_target: Optional[Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]]]
```

Comment:
Should these be added here and not be treated as model-specific args?

Reply:
they're in the base tokenizer class so I figured it's okay to put them here

lemme know if I should remove them

Reply:
Personally for me it's okey to have it here, as it is a general arg accepted by all tokenizers, though not used by all processors

Reply:
Sounds good, that will remove the need for special args for Udop so that's great :)